File: examples/tvos/objc/DownloadCache/RepositoryCell.m
//
//  RepositoryCell.m
//  RealmExamples
//
//  Created by Katsumi Kishikawa on 11/19/15.
//  Copyright © 2015 Realm. All rights reserved.
//

#import "RepositoryCell.h"

@implementation RepositoryCell

- (void)prepareForReuse {
    self.imageView.image = nil;
    self.titleLabel.text = nil;
}

@end

File: examples/tvos/swift/DownloadCache/RepositoryCell.swift
//
//  RepositoryCell.swift
//  RealmExamples
//
//  Created by kishikawakatsumi on 11/20/15.
//  Copyright © 2015 Realm. All rights reserved.
//

import UIKit

class RepositoryCell: UICollectionViewCell {
    @IBOutlet weak var avatarImageView: UIImageView!
    @IBOutlet weak var titleLabel: UILabel!

    override func prepareForReuse() {
        avatarImageView.image = nil
        titleLabel.text = nil
    }
}
